### DISCUSSION TITLE
Updating Pegs file to ICRU 90 from ICRU37

### DISCUSSION THREAD

**Discussion by karenrex1023** [nrc-cnrc/EGSnrc#1134] [Questions]
Hi,
I am still new to EGSnrc, and I am not too familiar with it yet. I've been running a CAVSPHnrc usercode in order to calculate a kwall correction factor for an A3 ionization chamber. The .pegs4dat file includes a 521ICRU for dry Air and C552, and I think that the values for air are based on ICRU37. I was wondering if you had any ideas of how the kwall would change with ICRU90 compared to ICRU37? I was also trying to figure out how I could make a pegs file with ICRU 90 data, or if there was one already available that I could download? I was a part of the EGSnrc course at UW-Madison last summer, and I noticed there was an option for PEGSless mode? I also wasn't sure if this was the direction I should try to go?

Best,
Karen Rex

**Comment by rtownson** [comment]
To use pegsless materials, just add something like this to the end of your input file. This is probably the right approach for you. You will need to add all the materials you want to use, you can't use both a pegs4dat file, and pegsless mode.

```
:start media definition:

    ### energy transport cutoffs
    ae = 0.521
    ap = 0.01
    ue = 10.511
    up = 10.0

    :start air:
        density correction file = air_dry_nearsealevel
    :stop air:

    :start water:
        density correction file = water_icru90
    :stop water:

:stop media definition:
```

This will use density correction files found in `HEN_HOUSE/pegs4/density_corrections`, unless you give the full path to a different file. Make sure to adjust `ae`, `ap`, etc. as appropriate for your source.

To create your own icru 90 c552, follow the instructions for creating a density correction file. Note that if you need to change the i-value away from the default, the ESTAR website may not do this properly (at least, it used to have a bug around this).

https://github.com/nrc-cnrc/EGSnrc/wiki/Generating-density-effect-correction-files-with-ESTAR

**Reply by karenrex1023** [reply]
Hi again,
 Just to clarify, the density correction file titled "air_dry_nearsealevel" is already ICRU90?
